src/graphql/query/myKiva.graphql

```diff
@@ -1,6 +1,52 @@
-query ftdQuery {
+query myKivaQuery {
 	my {
     id
+    loans(limit: 10) {
+      values {
+        id
+        name
+        gender
+        status
+        image {
+          id
+          url
+          hash
+        }
+        geocode {
+          city
+          state
+          country {
+            name
+            isoCode
+            region
+          }
+        }
+        loanAmount
+        loanFundraisingInfo {
```

Comment:
Once hasCompletedBorrowers is updated, I don't think this will be needed anymore, right? It would be good to remove it, because the current fundraising info is expensive to query.